interface/ceed-python/tests(will want to move eventually)/t100-vec.py

```diff
@@ -2,7 +2,7 @@
 # Test creation, setting, reading, restoring, and destroying of a vector
 
 import sys
-from libceed import MEM_HOST, USE_POINTER
+from libceed import CEED_MEM_HOST, CEED_USE_POINTER
```

Comment:
Do we want to try to make sure it's reasonable to do `from libceed import *` if people want?

Reply:
No, I'd say that's an anti-pattern, and it'll be cumbersome when people follow reasonable practices.

Reply:
What if these constants were declared as members inside the class `Ceed`? Would `Ceed.MEM_HOST` still feel clunky? In that case, we would't need to import each of the constants in every single file

Reply:
It should work to do `libceed.MEM_HOST', right?

Reply:
```python
from libceed import Ceed
ceed = Ceed('/cpu/self')
qf = ceed.QFunction(...)
qf.add_input('foo', 3, libceed.EVAL_INTERP)
```
doesn't work because `libceed` is not defined. This would work, and I think is entirely acceptable and simplest to implement:
```python
import libceed
ceed = libceed.Ceed('/cpu/self')
qf = ceed.QFunction(...)
qf.add_input('foo', 3, libceed.EVAL_INTERP)
```

Alternatively, we could make it an attribute of the relevant class/classes, e.g.,
```python
qf.add_input('foo', 3, ceed.EVAL_INTERP)
```
or
```python
qf.add_input('foo', 3, qf.EVAL_INTERP)
```